mopidy/core/mixer.py

```diff
@@ -99,3 +100,14 @@ def set_mute(self, mute):
             return result
 
         return False
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        return MixerState(volume=self.get_volume())
```

Comment:
Is it a conscious choice that we're just restoring volume and not mute state?

If not, I think the `volume` "coverage" should be changed to `mixer` to cover both volume and mute state.